src/components/themed/TransactionListRow.tsx

```diff
@@ -188,13 +188,14 @@ export function TransactionListRow(props: Props) {
     Share.open(shareOptions).catch(e => showError(e))
   })
 
+  // HACK: Handle 100% of the margins because of SceneHeader usage on this scene
   return (
     <CardUi4 icon={icon} onPress={handlePress} onLongPress={handleLongPress} marginRem={[0.5, 1]}>
       <SectionView dividerMarginRem={[0.2, 0.5]} marginRem={[0.25, 0]}>
         <>
           <View style={styles.row}>
             <EdgeText style={styles.titleText}>{name}</EdgeText>
-            <EdgeText style={styles.titleText}>{cryptoAmountString}</EdgeText>
+            <EdgeText style={styles.cryptoText}>{cryptoAmountString}</EdgeText>
```

Comment:
I don't know what this change is either. Provide task and screenshot

Reply:
![image (1)](https://github.com/EdgeApp/edge-react-gui/assets/517469/8e6249d8-3dfe-4323-9723-05d820c0288c)